compatibility/bazel_tools/create-daml-app/index.test.ts

```diff
@@ -307,7 +307,9 @@ test("log in as three different users and start following each other", async ()
 
   // Add Party 3 as well and check both are in the list.
   await follow(page1, party3);
-  await page1.waitForSelector(".test-select-following");
+  await page1.waitForFunction(
+    () => document.querySelectorAll(".test-select-following").length == 2
+  );
```

Comment:
Doesn't this make the length check below redundant?
```
   expect(followingList11).toHaveLength(2);
```

Reply:
Kind of but it seemed reasonable to keep the assertion separate from the check for how long we wait. Otherwise this is easy to get lost during refactoring.